cmasher/utils.py

```diff
@@ -233,6 +237,97 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
```

Comment:
this comment seems unnecessary since the docstring is right there.

Reply:
The reason for this comment is I'm new to `cmasher` so I was trying to follow the original style of it. And I noticed every function defined in `utils.py` has a comment on top in addition to the function docstring. Do you want me to remove it?